docs/imagecustomizer/how-to/sysext.md

```diff
@@ -0,0 +1,145 @@
+- Partitioning the image (root filesystem, verity, signature).
+- Applying integrity verification using dm-verity.
+- Signing the image (if needed).
+
+For detailed documentation on mkosi commands and configuration, refer to [doc](https://github.com/systemd/mkosi/blob/76377d3838635dd95beae6db393f06258582be2c/mkosi/resources/man/mkosi.1.md)
+
+## 1. Install mkosi
+
+Clone the latest version from GitHub (recommended):
+```  
+git clone https://github.com/systemd/mkosi.git
+cd mkosi
+./bin/mkosi --version
+```
+
+Or install via package manager.
+
+## 2. Define the Image Configuration
+
+1. Create a `mkosi.conf`: defines the build parameters for your sysext image. This configuration file tells mkosi what to include in the image, how to format it, and how to handle verification.
+
+2. Update `sysext.repart.d/` if needed: this directory contains partition definition files (.conf files) that instruct systemd-repart how to create and structure the partitions within the sysext image, including:
+    - Content partition (holds the actual files)
+    - Verity hash partition (used for integrity verification)
+    - Signature partition (optional, contains cryptographic signature)
+
+    Typically, users can create customized sysext.repart.d/ files for certain aspects of the image structure. However, when using Format=sysext - unlike Format=disk, you cannot completely replace these definitions with your own custom repart configurations. **For format=sysext, mkosi is specifically using its own predefined repart definitions located at [mkosi/resources/repart/definitions/sysext.repart.d](https://github.com/systemd/mkosi/tree/e4be13971af896ac6301b78cb045fb4cbe7a2b04/mkosi/resources/repart/definitions/sysext.repart.d). If you need to modify the partition structure (e.g., to change filesystem types or partition sizes), you'll need to edit the .conf files in the mkosi installation directory, which will be in the system paths if installed via package manager or in the cloned repository.**
+
+Example mkosi.conf
+
+```yaml
+[Output]
+Format=sysext
+ImageId=kubernetes
+ImageVersion=1.30.7
+OutputDirectory=mkosi.output
+
+[Validation]
+Verity=signed
+VerityKey=verity_key.pem
+VerityCertificate=verity_cert.pem
+
+[Content]
+ExtraTrees=/usr/local/bin/kubectl:/usr/local/bin/kubectl, /usr/local/bin/kubelet:/usr/local/bin/kubelet
+```
+
+### [Output] Section
+- Format=sysext → Specifies that we are building a system extension image.
+- ImageId=kubernetes → Defines the image identifier, which is used in the output filename.
+- ImageVersion=1.30.7 → Sets the version number of the image, included in metadata and the default filename.
+- OutputDirectory=mkosi.output → Specifies the directory where the output file will be stored.
+
+### [Validation] Section
+The Verity= setting determines how your sysext image is secured:
+- signed → Fully signed image with both hash data and a cryptographic signature (requires VerityKey & VerityCertificate).
```

Comment:
nit:

```Markdown
- `signed`: Fully signed image with both hash data and a cryptographic signature (requires VerityKey & VerityCertificate).
```

This is more consistent with how the existing docs are formatted. (I also think it's easier to read.)